docs/container.md

```diff
@@ -0,0 +1,313 @@
+---
+category: Develop
+previous: piwiks-ini-configuration
+next: plugin-settings
```

Comment:
This sounds good to me. Will look into it when I have the time.

> I'm not sure if it's beneficial to have 2 separate guides for this or whether it is confusing. We can still link between them so one can find it in 2 places which might be better. Not sure!

I think it's ok, some redundancy in the writing (specific to each use case) could provide a greater chance the reader will understand.